doc/user/radial_and_axial_expansion.rst

```diff
@@ -0,0 +1,86 @@
+
+.. math::
+    \frac{L_1 - L_0}{L_0} = \alpha(T_1)\left(T_1 - T_0\right).
+    :label: linearExp2
+
+Rearranging Equation :eq:`linearExp2`, we can obtain an expression for the new length, :math:`L_1`,
+
+.. math::
+    L_1 = L_0\left[1 + \alpha(T_1)\left(T_1 - T_0\right) \right].
+    :label: newLength
+
+Given Equation :eq:`linearExp2`, we can create expressions for the change in length between our "hot" temperature (Equation :eq:`hotExp`)
+
+.. math::
+    \begin{align}
+        \frac{L_h - L_0}{L_0} &= \alpha(T_h)\left(T_h - T_0\right),\\
+        \frac{L_h}{L_0} &= 1 + \alpha(T_h)\left(T_h - T_0\right).
+    \end{align}
+    :label: hotExp
+
+and "non-reference" temperature, :math:`T_c` (Equation :eq:`nonRefExp`),
+
+.. math::
+    \begin{align}
+        \frac{L_c - L_0}{L_0} &= \alpha(T_c)\left(T_c - T_0\right),\\
+        \frac{L_c}{L_0} &= 1 + \alpha(T_c)\left(T_c - T_0\right).
+    \end{align}
+    :label: nonRefExp
+
+These are used within ARMI to enable thermal expansion and contraction with a temperature not equal to the reference temperature, :math:`T_0`. By taking the difference between Equation :eq:`hotExp` and :eq:`nonRefExp`, we can obtain an expression relating the change in length, :math:`L_h - L_c`, to the reference length, :math:`L_0`,
+
+.. math::
+    \begin{align}
+        \frac{L_h - L_0}{L_0} - \frac{L_c - L_0}{L_0} &= \frac{L_h}{L_0} - 1 - \frac{L_c}{L_0} + 1, \\
+        &= \frac{L_h - L_c}{L_0}.
+    \end{align}
+    :label: diffHotNonRef
+
+Using Equations :eq:`diffHotNonRef` and :eq:`nonRefExp`, we can obtain an expression for the change in length, :math:`L_h - L_c`, relative to the non-reference temperature,
+
+.. math::
+    \frac{L_h - L_c}{L_c} &= \frac{L_h - L_c}{L_0} \frac{L_0}{L_c}\\
+    &= \left( \frac{L_h}{L_0} - \frac{L_c}{L_0} \right) \left( 1 + \alpha(T_c)\left(T_c - T_0\right) \right)^{-1}.
+    :label: expNewRelative
+
+Using Equations :eq:`hotExp` and :eq:`nonRefExp`, we can simplify Equation :eq:`expNewRelative`,
+
+.. math::
+    \frac{L_h - L_c}{L_c} = \frac{\alpha(T_h) \left(T_h - T_0\right) - \alpha(T_c)\left(T_c - T_0\right)}{1 + \alpha(T_c)\left(T_c - T_0\right)}.
+    :label: linearExpansionFactor
+
+Equation :eq:`linearExpansionFactor` is the expression used by ARMI in :py:meth:`linearExpansionFactor <armi.materials.material.Material.linearExpansionFactor>`.
+
+.. note::
+    :py:meth:`linearExpansionPercent <armi.materials.material.Material.linearExpansionPercent>` returns :math:`\frac{L - L_0}{L_0}` in %. The expression used in :py:meth:`linearExpansionFactor <armi.materials.material.Material.linearExpansionFactor>` can be found as follows,
+    
+    .. math::
+        \frac{L_h - L_c}{L_c} &= \frac{\frac{\alpha(T_h) \left(T_h - T_0\right)}{100} - \frac{\alpha(T_c)\left(T_c - T_0\right)}{100}}{\frac{1 + \alpha(T_c)\left(T_c - T_0\right)}{100}}, \\
+        &= \frac{\alpha(T_h) \left(T_h - T_0\right) - \alpha(T_c)\left(T_c - T_0\right)}{100 + \alpha(T_c)\left(T_c - T_0\right)}.
```

Comment:
I don't think writing out this math is really needed, as it's just multiplying and dividing by 100 and then simplifying that factor. Probably easier for the reader to save the text, IMO.

Reply:
https://github.com/terrapower/armi/pull/1204/commits/656cffbcf8170112b6d8f96d0b2c710d73115ef1